management_api_app/models/schemas/workspace_template.py

```diff
@@ -53,7 +53,10 @@ class Config:
                 "name": "My workspace template",
```

Comment:
Change example to be a docker image name (no spaces)
```suggestion
                "name": "my-tre-workspace",
```

Reply:
Good point